test_scripts/API/VehicleData/UnsubscribeVehicleData/005_Success_flow_2_apps.lua

```diff
@@ -0,0 +1,135 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TO ADD !!!
+-- Use case: TO ADD !!!
+-- Item: Use Case: request is allowed by Policies
+--
+-- Requirement summary:
+-- [UnsubscribeVehicleData] Mobile app wants to send a request to unsubscribe
+--  for already subscribed specified parameter
+--
+-- Description:
+-- In case:
+-- App1 is alredy subscribed to vehicle data
+-- App2 is alredy subscribed to vehicle data
+-- App2 sends valid UnsubscribeVehicleData to SDL
+-- This request is allowed by Policies and mobile app is subscribed for this parameter
+-- SDL must:
+-- not transfer this request to HMI
+-- Respond SUCCESS, success:true to mobile application
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local rpc_subscribe = {
+  name = "SubscribeVehicleData",
+  params = {
+    engineOilLife = true,
+    fuelRange = true,
+    tirePressure = true
+  }
+}
+
+local rpc_unsubscribe = {
+  name = "UnsubscribeVehicleData",
+  params = {
+    engineOilLife = true,
+    fuelRange = true,
+    tirePressure = true
+  }
+}
+
+local vehicleDataResults = {
+  engineOilLife = {
+    dataType = "VEHICLEDATA_ENGINEOILLIFE",
+    resultCode = "SUCCESS"
+  },
+  fuelRange = {
+    dataType = "VEHICLEDATA_FUELRANGE",
+    resultCode = "SUCCESS"
+  },
+  tirePressure = {
+    dataType = "VEHICLEDATA_TIREPRESSURE",
+    resultCode = "SUCCESS"
+  }
+}
+
+--[[ Local Functions ]]
+local function getVDParams()
+  local out = {}
+  for k in pairs(common.allVehicleData) do
+    table.insert(out, k)
+  end
+  return out
+end
+
+local function ptUpdateForApps(pTbl)
+  pTbl.policy_table.app_policies[common.getConfigAppParams(1).fullAppID].groups = { "Base-4", "Emergency-1" }
+  pTbl.policy_table.app_policies[common.getConfigAppParams(2).fullAppID].groups = { "Base-4", "Emergency-1" }
+  local grp = pTbl.policy_table.functional_groupings["Emergency-1"]
+  for _, v in pairs(grp.rpcs) do
+    v.parameters = getVDParams()
+  end
+  pTbl.policy_table.vehicle_data = nil
+end
+
+local function processRPCSubscribeSuccess(pAppID, pHMIsubscription)
```

Comment:
@GetmanetsIrina , please add parameter for RPC and use for different cases.
```suggestion
local function processRPCSubscribeSuccess(pAppID, pHMIsubscription, pRPCName)
```